packages/components/scripts/generateDSRComponents.ts

```diff
@@ -90,18 +89,26 @@ import { get${componentName}Css } from '${stylesBundleImportPath}';
         .replace(/\bbreakpoint\.l\b/, `'${breakpoint.l}'`) // inline breakpoint value from utilities-v2 for marque
         .replace(/{(isRequiredAndParentNotRequired\(.*)}/, '{/* $1 */}'); // comment out isRequiredAndParentNotRequired for now
 
+      if (hasSlot && !newFileContent.includes('FunctionalComponent')) {
+        newFileContent = newFileContent.replace(
+          /import { stripFocusAndHoverStyles } from '\.\.\/\.\.\/stripFocusAndHoverStyles';/,
```

Comment:
This could break easily since it is based on another very specific replacement. A save way would be to add it at the top or after the last import.